Makefile

```diff
@@ -1,10 +1,11 @@
-all     :; DAPP_LIBRARIES=' lib/dss-exec-lib/src/DssExecLib.sol:DssExecLib:0xfD88CeE74f7D78697775aBDAE53f9Da1559728E4' \
-           DAPP_BUILD_OPTIMIZE=1 DAPP_BUILD_OPTIMIZE_RUNS=1 \
-           dapp --use solc:0.6.12 build
-clean   :; dapp clean
-           # Usage example: make test match=SpellIsCast
-test    :; ./test-dssspell.sh $(match)
-deploy  :; make && dapp create DssSpell | xargs ./verify.py DssSpell
-estimate:; ./estimate-deploy-gas.sh
-flatten :; hevm flatten --source-file "src/DssSpell.sol" > out/flat.sol
-archive-spell :; ./archive-dssspell.sh $(date)
+all             :; DAPP_LIBRARIES=' lib/dss-exec-lib/src/DssExecLib.sol:DssExecLib:0xfD88CeE74f7D78697775aBDAE53f9Da1559728E4' \
+                    DAPP_BUILD_OPTIMIZE=1 DAPP_BUILD_OPTIMIZE_RUNS=1 \
+                    dapp --use solc:0.6.12 build
+clean           :; dapp clean
+                    # Usage example: make test match=SpellIsCast
+test            :; ./test-dssspell.sh $(match)
+test-dev        :; ./test-dev-dssspell.sh $(match)
```

Comment:
nit: consider `test-noop` or `test-noopts`
cc: @brianmcmichael

Reply:
those make me think of no op, as in a function that fails, but maybe that is just me

Reply:
yeah, maybe `noopts` is more descriptive, as you prefer.